docs/sources/writing-guide/contribute-release-notes/index.md

```diff
@@ -12,25 +12,130 @@ keywords:
+
+   > **Note:** A Product Manager (PM) is responsible for update the version number of that channel after a release goes out. 
+   
+1. The technical writer posts in the Slack channel that the What's New PR is ready for contributions. 
+1. The PM or PMM on point communicates with eng squads without a PM resource about who’s on point for What’s New PR reviews from within the PM and PMM orgs.
+
+   Refer to [Grafana Releases and Release Dates](https://grafana-intranet--simpplr.visualforce.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000001AXrwAAG) for a list of release dates and go-to-market coordinators.
+
+### Throughout a release
+
+A Grafana release cycle is typically 6-8 weeks, and during that time, developers are empowered to add to the What's New document incrementally. To see a list of release dates and assigned PMs and PMMs, refer to [Grafana Releases and Release Dates](https://grafana-intranet--simpplr.visualforce.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000001AXrwAAG).
+
+Complete the following steps throughout a release cycle:
+
+1. Dev teams add commits directly against the branch.
+
+   > **Note:** As an alternative, Dev teams can cut PRs against the branch, add What’s New content, collaborate with Reviewers, and merge the PR into the What's New branch.
+
+1. The technical writer periodically checks the What's New PR to determine how much content has been added.
+
+At this point, the larger Marketing org can review the What’s New content in the branch.
+
+### At the end of a release
+
+Complete the following steps in the days leading up to the release:
+
+1. Five days before the GA release, the PM or PMM who is running the release sends out a **Last Call** to dev teams to get their What’s New content in.
+
+1. Two days before the release, What’s New content is frozen.
+
+1. On release day, the technical writer merges the What’s New branch into main and backports the doc to the feature version branch.
+
+1. A PM reviews all content and adjusts wording, where appropriate.
+
+1. The technical writer provides a copy edit.
+
+1. The technical writers coordinates with Release Management and the GTM team for precise timing of when to merge the What's New doc into main.
+
+## How to determine if content belongs in a What's New document
+
+Grafana publishes a What’s New [docs page](//docs/grafana/latest/whatsnew/) and [blog post](https://grafana.com/blog/2022/11/29/grafana-9.3-release/) along with every minor and major release. These posts are popular, and a good way for users to learn about the exciting new things we’ve released. What’s New also drives our go-to-market enablement: we train the field and make videos on the topics in What’s New. However, unlike our comprehensive changelog, What’s New is curated. If it contained every update it would be too noisy for people to read, and if we wrote a detailed What’s New post for every little bug fix, we’d be wasting our time.
```

Comment:
Just breaking this up to improve readability 

```suggestion
Grafana publishes a What’s New [docs page](//docs/grafana/latest/whatsnew/) and [blog post](https://grafana.com/blog/2022/11/29/grafana-9.3-release/) along with every minor and major release. These posts are popular, and a good way for users to learn about the exciting new things we’ve released. What’s New also drives our go-to-market enablement: we train the field and make videos on the topics in What’s New. 

However, unlike our comprehensive changelog, What’s New is curated. If it contained every update, it would be too noisy for people to read, and if we wrote a detailed What’s New post for every little bug fix, we’d be wasting our time.
```